lib/mnesia/test/mnesia_trans_access_test.erl

```diff
@@ -463,8 +467,191 @@ select14(Config) when is_list(Config) ->
     Test(Tab2),
     Test(Tab3),
     Test(Tab4),
+
+    OneRec = {Tab1, 1, 2},
+    TwoRec = {Tab1, 2, 3},
+    All = [OneRec, TwoRec],
+    AllPat = [{'_', [], ['$_']}],
+    ?match(All, rpc:call(Node1, mnesia, ets, [fun() -> Loop(Tab1, AllPat) end])),
+
+    ?verify_mnesia(Nodes, []).
+
+%% select_reverse
+select_reverse(suite) -> [];
+select_reverse(Config) when is_list(Config) ->
+    [Node1] = Nodes = ?acquire_nodes(1, Config),
+    Tab = select_reverse_tab,
+    Schema = [{name, Tab}, {attributes, [k, v]}, {ram_copies, [Node1]}],
+    ?match({atomic, ok},  mnesia:create_table(Schema)),
+
+    OneRec = {Tab, 1, 2},
+    TwoRec = {Tab, 2, 3},
+    ThreeRec = {Tab, 3, 4},
+    OnePat = [{{Tab, '$1', 2}, [], ['$_']}],
+    All = [ThreeRec,TwoRec,OneRec],
+    AllPat = [{'_', [], ['$_']}],
```

Comment:
It seems you don't use those variables? Did you want to test anything more here, or they should be removed?

Reply:
ah I think I moved some things to `select_reverse14` and forgot this, I'll remove it here, thanks